### PR TITLE
cruft update and loose dependencies in pyproject.toml

### PR DESCRIPTION
**Pull request by erichesse**
# Changes
- update to a782ed cruft template

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,11 +9,16 @@ urls = { Repository = "https://github.com/robert-koch-institut/mex-model" }
 requires-python = "<3.13,>=3.11"
 dependencies = []
 optional-dependencies.dev = [
-    "black==24.8.0",
-    "ipdb==0.13.13",
-    "myst-parser==4.0.0",
-    "ruff==0.5.7",
-    "sphinx==7.4.7",
+    "black==24.8.0,<25",
+    "ipdb==0.13.1,<1",
```
Comment (mr-kamran-ali) [inline pyproject.toml:13]: it should be greater than `>=` for black, ipdb and myst-parser as well. 

z.b: `"ipdb>=0.13.13,<1",`

Comment (mr-kamran-ali) [reply pyproject.toml:13]: also check if `black` is still needed. It was removed from the template and some other repos: https://github.com/robert-koch-institut/mex-template/blob/a782ed7ea131e6d0cd67d4cc81975f2328228bcd/mex-%7B%7B%20cookiecutter.project_name%20%7D%7D/pyproject.toml#L11

File: pyproject.toml [hunk 2]
```diff
@@ -9,11 +9,16 @@ urls = { Repository = "https://github.com/robert-koch-institut/mex-model" }
...
+    "myst-parser==4.0.0,<5",
+    "mypy>=1.11.0,<2",
+    "pytest-cov>=5.0.0,<6",
+    "pytest-random-order>=1.1.1,<2",
+    "pytest-xdist>=3.6.1,<4",
+    "pytest>=8.3.1,<9",
```
Comment (rababerladuseladim) [inline pyproject.toml:19]: This repo has neither type checking nor tests.

```suggestion
```

**Review by rababerladuseladim — CHANGES_REQUESTED** [review]
remove unnecessary packages

File: pyproject.toml (hunk 2 above)
Comment (erichesse) [reply pyproject.toml]: https://github.com/robert-koch-institut/mex-model/pull/107/commits/30678238ff52cda995e040f4dbecf8b2371e55fa

File: pyproject.toml (hunk 1 above)
Comment (erichesse) [reply pyproject.toml]: https://github.com/robert-koch-institut/mex-model/pull/107/commits/30678238ff52cda995e040f4dbecf8b2371e55fa

**Review by mr-kamran-ali — APPROVED** [review]

**Review by rababerladuseladim — APPROVED** [review]

**Pull request merged**